packages/components-library/tests/unit/filters/FilterCard.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { mount, createLocalVue } from '@vue/test-utils'
+import FilterCard from '@/components/filters/FilterCard.vue'
+import { BootstrapVue } from 'bootstrap-vue'
+
+const localVue = createLocalVue()
+localVue.use(BootstrapVue)
+
+const name = 'name'
+
+describe('FilterCard.vue', () => {
+  describe('starts closed', () => {
+    const wrapper = mount(FilterCard, { localVue, propsData: { name } })
+    it('can open and close', async () => {
+      expect(wrapper.find('div#name').element.style.display).toEqual('none') // Closed
+      await wrapper.find('.card-header').trigger('click')
+      expect(wrapper.find('div#name').element.style.display).toEqual('') // Open
+      await wrapper.find('.card-header').trigger('click')
+      expect(wrapper.find('div#name').element.style.display).toEqual('none') // Closed again
+    })
+  })
+
+  describe('starts open', () => {
```

Comment:
maybe, rephrase with: 'starting open' ? starting open...it can open and close (for instance)